appveyor.yml

```diff
@@ -20,8 +25,9 @@ shallow_clone: true
 clone_depth: 1
 test_script:
   - node --version
-  - npm --version
+  - yarn --version || npm --version
```

Comment:
I believe you are correct in your understanding of `||`. `yarn --version` exits with error on node versions less than 4: ([appveyor build](https://ci.appveyor.com/project/boneskull/mocha/build/809)). We could do 
```
- yarn --version || true
- npm --version
```
in order to avoid the error and still print both